containers/Dockerfile-Alpine_Musl

```diff
@@ -23,8 +23,8 @@ WORKDIR /keysinuse-openssl/packaging
 RUN make /keysinuse-openssl/bin/keysinuseutil
 
 # Copy files to staging
-RUN mkdir /usr/lib/keysinuse
-RUN cp /keysinuse-openssl/bin/keysinuse.so            /usr/lib/keysinuse
-RUN cp /keysinuse-openssl/bin/keysinuseutil           /usr/lib/keysinuse
-RUN cp /keysinuse-openssl/packaging/common/postinst    /usr/lib/keysinuse/installkeysinuse
-RUN chmod 0755 /usr/lib/keysinuse/installkeysinuse
\ No newline at end of file
+RUN mkdir /keysinuse
```

Comment:
Why in the root directory? /usr/lib might not be the right place, but I don't know if the root directory is either.